examples/pwa/pwa.html

```diff
@@ -11,6 +11,33 @@
     if (window.location.pathname.indexOf('/examples') == 0) {
       window.location.assign('/pwa');
     }
+
+    function loadScript(url, callback) {
+        // Adding the script tag to the head as suggested before
+        var head = document.getElementsByTagName('head')[0];
+        var script = document.createElement('script');
+        script.type = 'text/javascript';
+        script.src = url;
+
+        // Then bind the event to the callback function.
+        // There are several events for cross browser compatibility.
+        script.onreadystatechange = callback;
+        script.onload = callback;
+
+        // Fire the loading
+        head.appendChild(script);
+    }
+
+    // Using this explicitly, as firefox errors due to our polyfills
+    loadScript("https://cdnjs.cloudflare.com/ajax/libs/webcomponentsjs/1.0.3/webcomponents-sd-ce.js", function() {
+      console.log('Started loading webcomponents polyfill!');
+    });
+
+    setTimeout(function() {
```

Comment:
The reason why I put it in a timeout, was to:

(a) visually see our JS be applied to the page vs. just the web components poly fill

(b) I remember you saying that we were going to have developers ensure that we are loaded after their polyfill

For the case of (b), would you still like this to be the case? I am sure that they can both run simultaneously and we will still work as expected. Or, would you like a function in `pwa.js` to wait for an event to be fired from the webcomponents `loadscript()` call?